### PR TITLE
<feat> bone structure of SPA ready(vanilla js)

### PR DESCRIPTION
**Pull request by ynslee**
Pong-related frontend files and package.json moved in to /frontend/pong folder.
SPA vanilla js bone structure and routing structure is ready. css is at the moment, most minimal as possible.

### CODE REVIEW

**Review by rosamakinen — APPROVED** [review]
I dont think we need the .vscode folder, but other than that looks good.

File: frontend/package-lock.json
Comment (Stte) [inline frontend/package-lock.json:1]: I think this file should be included.

https://stackoverflow.com/a/44210813

File: .vscode/configurationCache.log
Comment (Stte) [inline .vscode/configurationCache.log:1]: .vscode to gitignore

**Review by Stte — CHANGES_REQUESTED** [review]
Very nice! Me like :+1:   Two very minor nitpicks (check comments). Other than that all good! :D

File: frontend/package-lock.json
Comment (ynslee) [reply frontend/package-lock.json:1]: package-lock.json added now! it is not in seperate files for pong folder and frontend folder

**Review by Stte — APPROVED** [review]
LGTM :+1:

**Pull request merged**